lib/client/api.go

```diff
@@ -446,6 +446,9 @@ func Status(profileDir string, proxyHost string) (*ProfileStatus, []*ProfileStat
 		if file.IsDir() {
 			continue
 		}
+		if file.Mode()&os.ModeSymlink != 0 {
```

Comment:
Comment explaining why symlinks are skipped because it's not clear from reading this function that you are skipping over aliases.

```go
// Skip over symlinks (aliases) to prevent double printing of profiles.
```